examples/l1-resources/stream-instance.ts

```diff
@@ -0,0 +1,33 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { CfnStreamInstance, StreamsDataProcessRegionCloudProvider } from 'awscdk-resources-mongodbatlas'
+import { env } from 'node:process';
+
+const PROFILE = "default-oriol";
+const PROJECT_ID = "65dca009938ec8392b94e7e6";
+const INSTANCE_NAME = "testInstanceName";
```

Comment:
would favor the structure defined in other examples of receiving certain user specific values as parameters in the context and not have them hardcoded.

Reply:
Makes sense, changed